packages/@ourworldindata/components/src/styles/mixins.scss

```
@@ -36,6 +36,15 @@
     }
 }
 
+@mixin dod-span {
+    border-bottom: 1px dotted #1d3d63;
```

Can we replace this with the color name?
```suggestion
    border-bottom: 1px dotted $blue-90;
```